docs/quickstart.rst

```diff
@@ -159,8 +159,8 @@ Task 3: Persist service configuration for use in multiple sessions
     ...         remote: my_server
     ... """)
 
-    >>> %%presto_local
```

Comment:
This was actually correct. By default, duct magic functions are registered when you use `register_from_config`. Can you drop this patch?